Gopkg.toml

```diff
@@ -0,0 +1,30 @@
+
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/miekg/pkcs11"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/thalesignite/crypto11"
```

Comment:
Got it!

`dep init` doesn't want to write a `Gopkg.toml` that lists the repo itself. But it seems that it does a case-insensitive comparison. So if you clone `github.com/ThalesIgnite/crypto11`, it doesn't notice that the demo's `import github.com/thalesignite/crypto11` is actually the same thing.

Per https://blog.golang.org/package-names Go package names are lower case, so I think the answer is we need to stop with the camelcase in ThalesIgnite. I'll adjust the `README.md` accordingly, and fix up the `.toml` files.